pkg/carvelhelpers/fetcher.go

```diff
@@ -36,40 +30,19 @@ func GetImageDigest(imageWithTag string) (string, string, error) {
 	return NewImageOperationsImpl().GetImageDigest(imageWithTag)
 }
 
-// newRegistry returns a new registry object by also
-// taking into account for any custom registry or proxy
-// environment variable provided by the user
-func newRegistry() (registry.Registry, error) {
-	verifyCerts := true
-	skipVerifyCerts := os.Getenv(constants.ConfigVariableCustomImageRepositorySkipTLSVerify)
-	if strings.EqualFold(skipVerifyCerts, "true") {
-		verifyCerts = false
-	}
-
+// newRegistry returns a new registry object by also taking
+// into account for any custom registry provided by the user
+func newRegistry(registryName string) (registry.Registry, error) {
```

Comment:
Should we also rename all occurrences of `registryName` to `host`? `registryName` seems confusing.

Reply:
I see the go-containerregistry [project](https://github.com/google/go-containerregistry/blob/main/pkg/name/registry.go#LL114C12-L114C12) is also using the registry name to store the registry host along with port. To avoid confusion we can use `registryHost` instead of `registryName`.